force-app/main/default/classes/ServiceScheduleService_TEST.cls

```diff
@@ -134,7 +134,8 @@ public with sharing class ServiceScheduleService_TEST {
             DaysOfWeek__c = '1;2;3;4;5;6;7',
             PrimaryServiceProvider__c = primaryServiceProvider.Id,
             OtherServiceProvider__c = otherServiceProvider.Id,
-            CreateServiceSessionRecords__c = true
+            CreateServiceSessionRecords__c = true,
+            MonthlyRecurrenceOption__c = '+1MO'
```

Comment:
Same, no `+` at this point?

Reply:
copy/paste error. Will remove that. Thanks for catching it